app/src/main/kotlin/no/nav/tiltakspenger/vedtak/repository/søknad/SøknadDAO.kt

```diff
@@ -206,6 +219,7 @@ internal object SøknadDAO {
         val vedlegg = int("vedlegg")
         val sakId = SakId.fromString(string("sak_id"))
         val saksnummer = Saksnummer(string("saksnummer"))
+        val oppgaveId = stringOrNull("oppgave_id")?.toInt()
```

Comment:
Litt usikker på hvor jeg skal kommentere dette, men jeg foreslår at vi legger en OppgaveId-type i libs. Vanligvis bruker jeg lagre eksterne IDer som String, men her skal vi sende den tilbake til oppgave-APIet og hvis de krever at den er number, synes jeg det er greit og ha den som int inne i OppgaveId-typen og basen.

Dvs. noe som

data class OppgaveId(private val value: String)
eller
data class OppgaveId(private val value: Int)

Her kan man og ha en value class hvis man føler for det.